src/Examine.AzureDirectory/AzureDirectoryFactory.cs

```diff
@@ -45,7 +45,7 @@ public AzureDirectoryFactory(bool isReadOnly)
         /// <returns>
         /// The <see cref="Lucene.Net.Store.Directory"/>.
         /// </returns>
-        public override Lucene.Net.Store.Directory CreateDirectory(DirectoryInfo luceneIndexFolder)
+        public  Lucene.Net.Store.Directory CreateDirectory(DirectoryInfo luceneIndexFolder)
```

Comment:
Should this just be an explicit implementation of this? I'll merge and update, Thanks!

Reply:
see rev 018e1c7f33ea0fb828cef0725d5728042798cf04